app/helpers/interview_questions_helper.rb

```diff
@@ -0,0 +1,13 @@
+module InterviewQuestionsHelper
+  def color_link(question)
+    color = "bg_link"
+    color += " bg_" + question.mark unless question.mark.blank?
+    color
+  end
+
+  def button_next(interview, question)
+    next_but = "NEXT"
+    next_but = "FINISH" if (interview.interview_questions.count - 1 + interview.interview_questions.first.id) == question.id
```

Comment:
view layer logic is housed in a helper, business layer logic is housed in a model